server/app/assets/stylesheets/uswds/styles.scss

```diff
@@ -0,0 +1,3 @@
+@forward 'uswds-theme';
+@forward 'uswds';
```

Comment:
Good question!  Let me explain.  `@forward` is a Sass annotation that behaves like an import statement would in JS.  [Sass docs](https://sass-lang.com/documentation/at-rules/forward/).  So, when Webpack starts converting the Sass and bundling our CSS, it starts with the `app/assets/stylesheets/uswds/styles.scss` as the entry point (See the `webpack-config.js` entrypoint list) includes all the styles from the referenced files (`uswds-theme.scss`, `uswds-theme-custom-styles.scss` and even `node_modules/@uswds/uswds/dist/scss/stylesheets/uswds.scss`).  It is able to find that last file in the node modules because we told it to search this path in the sass-loader configuration in `webpack-config.js`:
![image](https://github.com/civiform/civiform/assets/134556723/5cd56f32-cbf8-4f59-81be-b5405903bf7b)

Reply:
Gotcha! Thank you for explaining :)